packages/datadog-core/test/storage.spec.js

```diff
@@ -16,17 +16,17 @@ describe('storage', () => {
   })
 
   afterEach(() => {
-    testStorage('legacy').enterWith(undefined)
+    testStorage.enterWith(undefined)
```

Comment:
It's not clear to me, from the PR description, why these changes are included in this PR?